scpdt/plugin.py

```diff
@@ -159,6 +162,58 @@ def collect(self):
 
 def _get_parser():
     """
-    Return instance of DTParser with default configuration
+    Return instance of DTParser
+    """
+    return DTParser(config=dt_config)
+
+
+def _get_runner(checker, verbose, optionflags):
+    import doctest
+    """
+    Override function to return instance of DebugDTRunner
     """
-    return DTParser(config=dt_config)
\ No newline at end of file
+    class PytestDTRunner(DebugDTRunner):
+
+        def __init__(self, checker=None, verbose=None, optionflags=None, config=None, continue_on_failure=True):
+            super().__init__(checker=checker, verbose=verbose, optionflags=optionflags, config=config)
+            """
+            If `continue_on_failure is True, failures and exceptions are collected in the out stream. 
+            If it's False, the failure or exception is raised immediately, potentially stopping further 
+            execution of the particular doctest
+            """
+            self.continue_on_failure = continue_on_failure
+
+        def run(self, test, compileflags=None, out=None, clear_globs=False):
+            """
+            Run tests in context managers:
+            np_errstate: restores the numpy errstate and printoptions when done
+            user_context_manager: allows user to plug in their own context manager
+            matplotlib_make_nongui: temporarily make the matplotlib backend non-GUI
+            """
```

Comment:
Please refer to `testmod` source for a discussion of the order of context managers.

Reply:
Can you please clarify what you mean by this? Do you want me to clean the comment or replace it with the comment in `testmod`?

Reply:
Either way